Sources/Flows/OAuth2.swift

```diff
@@ -106,11 +106,16 @@ open class OAuth2: OAuth2Base {
 		
 		didAuthorizeOrFail = callback
 		logger?.debug("OAuth2", msg: "Starting authorization")
-		tryToObtainAccessTokenIfNeeded(params: params) { successParams in
+		tryToObtainAccessTokenIfNeeded(params: params) { successParams, error in
 			if let successParams = successParams {
 				self.didAuthorize(withParameters: successParams)
 			}
 			else {
+				self.logger?.debug("OAuth2", msg: "Error obtaining token \(String(describing: error))")
```

Comment:
`tryToObtainAccessTokenIfNeeded` already logs the error, so this here will double-log the same error. I'd remove this one or use it to replace the less verbose one on line 194 (now 199) – could say "error refreshing token".